src/mrpro/utils/__init__.py

```diff
@@ -10,6 +11,8 @@
 
 __all__ = [
     "broadcast_right",
+    "fill_range_",
+    "modify_acq_info",
```

Comment:
where is this coming from?

Reply:
and more importantly, why does precommit not remove it??